DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/CampaignResultsAlgorithmServlet.java

```diff
@@ -32,10 +32,13 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.HashMap;
 
 // obtains estimation results for all the pending DSA campaigns
-@WebServlet("/data")
-public class DSACampaignDataServlet extends HttpServlet {
+@WebServlet("/estimation-results")
+public class CampaignResultsAlgorithmServlet extends HttpServlet {
```

Comment:
Can we rename to CampaignEstimationResultsServlet

Reply:
Done